templates/edit_match_result.html

```diff
@@ -26,16 +26,43 @@
 <div id="scoreTemplate" style="display: none;">
   <div class="well well-{{"{{alliance}}"}}">
     <div class="form-group">
-      <label>Autonomous</label>
-      <input name="{{"{{alliance}}"}}AutoPoints" class="form-control"/>
+      <label>Taxi</label>
+      <div class="status-well">
+        <label>Taxi 1</label>
+        <input name="{{"{{alliance}}"}}Taxi1"/>
+        <label>Taxi 2</label>
+        <input name="{{"{{alliance}}"}}Taxi2"/>
+      </div>
     </div>
-    <div class="form-group">
-      <label>Teleoperated</label>
-      <input name="{{"{{alliance}}"}}TeleopPoints" class="form-control"/>
+    <div class="status-well">
+      <label>Shelf</label>
+      <div class="form-group">
+        <label>AutonBottom</label>
+        <input name="{{"{{alliance}}"}}AutonBottom"/>
+        <label>AutonTop</label>
+        <input name="{{"{{alliance}}"}}AutonTop"/>
+        <br/>
+        <label>TeleopBottom</label>  
+        <input name="{{"{{alliance}}"}}TeleopBottom"/>
+        <label>TeleopTop</label>  
+        <input name="{{"{{alliance}}"}}TeleopTop"/>
+        <br/>
+        <label>Golden Cube</label>  
+        <input name="{{"{{alliance}}"}}GoldenCube"/>
```

Comment:
Maybe for the boolean-valued fields, it would be more ergonomic to use `<input type="checkbox">`?